### PR TITLE
Make Dev == Live evaluate to almost true :)

### PR DESCRIPTION
**Pull request by ldb**

### CODE REVIEW

**Review by akkbng — APPROVED** [review]
LGTM

File: kratos/values-live.yaml [hunk 1]
```diff
@@ -332,17 +332,28 @@ kratos:
 
       flows:
         registration:
-          ui_url: https://auth-example.api.live.mindtastic.lol/registration
+          lifespan: 15m
+          ui_url: https://auth-example.api.dev.mindtastic.lol/registration
```
Comment (jgraeger) [inline kratos/values-live.yaml:336]:
```suggestion
          ui_url: https://auth-example.api.live.mindtastic.lol/registration
```

File: kratos/values-live.yaml [hunk 2]
```diff
@@ -332,17 +332,28 @@ kratos:
...
+          after:
+            password:
+              hooks:
+                - hook: session
+
         login:
-          ui_url: https://auth-example.api.live.mindtastic.lol/login
+          lifespan: 15m
+          ui_url: https://auth-example.api.dev.mindtastic.lol/login
```
Comment (jgraeger) [inline kratos/values-live.yaml:344]:
```suggestion
          ui_url: https://auth-example.api.live.mindtastic.lol/login
```

File: kratos/values-live.yaml [hunk 3]
```diff
@@ -332,17 +332,28 @@ kratos:
...
         error:
-          ui_url: https://auth-example.api.live.mindtastic.lol/error
+          ui_url: https://auth-example.api.dev.mindtastic.lol/error
```
Comment (jgraeger) [inline kratos/values-live.yaml:346]:
```suggestion
          ui_url: https://auth-example.api.live.mindtastic.lol/error
```

File: kratos/values-live.yaml [hunk 4]
```diff
@@ -332,17 +332,28 @@ kratos:
...
         logout:
           after:
-            default_browser_return_url: https://auth-example.api.live.mindtastic.lol
+            default_browser_return_url: https://auth-example.api.dev.mindtastic.lol
```
Comment (jgraeger) [inline kratos/values-live.yaml:349]:
```suggestion
            default_browser_return_url: https://auth-example.api.live.mindtastic.lol
```

**Review by jgraeger — APPROVED** [review]
Approved but apply the requested changes

**Pull request merged**